<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCSCClass</class>
 <widget class="QWidget" name="MCSCClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>325</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Minecraft要塞坐标计算器</string>
  </property>
  <widget class="QTabWidget" name="stepMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>360</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_start">
    <attribute name="title">
     <string>开始</string>
    </attribute>
    <widget class="QLabel" name="mainPageTitle_start">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>580</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(155, 183, 212);</string>
     </property>
     <property name="text">
      <string>  开始</string>
     </property>
    </widget>
    <widget class="QLabel" name="pageInfoTitle_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>560</width>
       <height>200</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>你需要投掷末影之眼，然后按要求记录投掷时的坐标与
其飞行路径水平延伸线尽头的坐标。
请尽可能保证坐标准确。
另外，你需要保证所有末影之眼均指向同一座要塞。
准备好三个以上末影之眼，然后开始吧！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_start">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QComboBox" name="language">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>262</y>
       <width>200</width>
       <height>36</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>简体中文</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="title_language">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>111</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Language:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="title_translator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>295</y>
       <width>310</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_inputPos">
    <attribute name="title">
     <string>键入坐标</string>
    </attribute>
    <widget class="QLabel" name="mainPageTitle_input">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>580</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(155, 183, 212);</string>
     </property>
     <property name="text">
      <string>  输入坐标</string>
     </property>
    </widget>
    <widget class="QLabel" name="pageInfoTitle_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>560</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[提示文本]
如果未显示提示文本，则语言文件可能有误。</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="throwPos_X">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="prefix">
      <string>X </string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-65536.000000000000000</double>
     </property>
     <property name="maximum">
      <double>65536.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="title_input_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>投掷时坐标</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="throwPos_Y">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="prefix">
      <string>Y </string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-65536.000000000000000</double>
     </property>
     <property name="maximum">
      <double>65536.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="title_input_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>延线终坐标</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="landingPos_X">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="prefix">
      <string>X </string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-65536.000000000000000</double>
     </property>
     <property name="maximum">
      <double>65536.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="landingPos_Y">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="prefix">
      <string>Y </string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-65536.000000000000000</double>
     </property>
     <property name="maximum">
      <double>65536.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_nextInput">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ctr_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>565</width>
       <height>90</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
</string>
     </property>
     <property name="text">
      <string>Click here to read the translation</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_calibration">
    <attribute name="title">
     <string>校准</string>
    </attribute>
    <widget class="QLabel" name="mainPageTitle_calibration">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>580</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(155, 183, 212);</string>
     </property>
     <property name="text">
      <string>  校准</string>
     </property>
    </widget>
    <widget class="QLabel" name="pageInfoTitle_calibration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>560</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[提示文本]
如果未显示提示文本，则语言文件可能有误。</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_goResult">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>查看结果</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_goCalibration">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>校准</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ctr_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>565</width>
       <height>90</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
</string>
     </property>
     <property name="text">
      <string>Click here to read the translation</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_result">
    <attribute name="title">
     <string>结果</string>
    </attribute>
    <widget class="QLabel" name="mainPageTitle_result">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>580</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(155, 183, 212);</string>
     </property>
     <property name="text">
      <string>  结果</string>
     </property>
    </widget>
    <widget class="QLabel" name="pageInfoTitle_result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>560</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>总参考线：-条
可能的要塞坐标：(---,---)
最大误差：---</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
